README.md

```diff
@@ -0,0 +1,285 @@
+# Duxy
+
+[![npm version](https://img.shields.io/npm/v/duxy.svg)](https://www.npmjs.com/package/duxy) [![License](https://img.shields.io/npm/l/duxy.svg)](https://www.npmjs.com/package/duxy) [![Build Status](https://travis-ci.org/producthunt/duxy.svg)](https://travis-ci.org/producthunt/duxy)
+
+RESTful resources.
+
+## Table of Contents
+
+  1. [Installation](#installation)
+  1. [Usage](#usage)
+    1. [Setup](#setup)
+    1. [Usage](#usage-1)
+    1. [DSL](#dsl)
+      1. [get](#getname--path-)
+      1. [post](#postname--path-)
+      1. [put](putname--path-)
+      1. [patch](#patchname--path-)
+      1. [del](#delname--path-)
+      1. [namespace](#namespacename--path--fn)
+      1. [resources](#resourcesname--path-only-findone-findall-create-update-delete-fn)
+      1. [resource](#resourcename--path-only-findone-create-update-delete--fn)
+    1. [Adapter](#adapter)
+  1. [Development](#development)
+  1. [Contributing](#contributing)
+  1. [License](#license)
+
+## Installation
+
+```
+$ npm install duxy --save
+```
+
+## Usage
+
+#### Setup
+
+```js
+// api.js
+import duxy from 'duxy';
+import duxySuperagent from 'duxy-superagent';
+import request from 'superagent';
+
+const http = duxySuperagent(superagent)();
+
+export default duxy({ http }, ({ get, resources }) => {
+  get('about');
+
+  resources('posts', { only: ['findAll', 'findOne'] }, () => {
+    resource('followers', { only: ['create', 'delete'] });
+  });
+});
+```
+
+#### Usage
+
+```js
+import api from 'ph/api';
+
+const { body } = api.about()                      // GET /about
+const { body } = api.posts.findOne({ id: 1 })     // GET /posts/1
```

Comment:
```
const { body } = api.posts.findOne({ id: 1 })     // GET /posts/1
```

This technically does a `GET /posts/1?id=1` right now (even though that's a bug)